### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by heyjinjs**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/heyjinjs/test/pull/1" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Ember-castle 
+# Ember Castle
```
Comment (heyjinjs) [inline README.md:1]: like

Comment (heyjinjs) [inline README.md:1]: why

Comment (heyjinjs) [inline README.md:1]: !fix

**Pull request merged**